runtime/meta.c

```diff
@@ -184,7 +186,7 @@ CAMLprim value caml_realloc_global(value size)
     for (i = actual_size; i < requested_size; i++){
       Field (new_global_data, i) = Val_long (0);
     }
-    caml_global_data = caml_check_urgent_gc(new_global_data);
```

Comment:
Why removing this call?

Reply:
Actually this one has calls to caml_initialize as well (a few lines above). But again I am assuming that it was correct before you added the line during your work on statmemprof. I can also leave check_urgent_gc in place independently of checking for callbacks.

Reply:
Same as above.

Reply:
Done.